### ISSUE TITLE
Relate fields in 8.2.0  not listing  when a module's  "List " display is selected. In 8.1.3 they list correctly.

### ISSUE THREAD

**Issue by larrymo** [salesagility/SuiteCRM-Core#181]
#### Issue
This issue is present in several modules.

I modified the "Calls" to add two relate fields.  One I called PtrPerson, the other PtrCompany.  These were relate fields to Contact and Accounts respectively.  I modified the edit and list modules to display these.

Account and contact information was entered and displayed correctly proving the information was entered and saved.

IN 8.2.0 ,  when going to the Call module, and selecting "List"  the Relate information was not displayed.

IN 8.1.3,  it was displayed.

#### Expected Behavior
I expected Suitecrm to display fields linked via Relate to behave themselves and be displayed.

#### Context

<!--- How has this bug affected you? ..   This bug broke my entire database which uses the listing of  Relate fields extensively.

Ive tested this extensively.  Suitecrms with version =<8.1.3 don't seem to be affected. Versions =>8.2.0 are.

Ive checked this on virtual machines using linuxmint, ubuntu, debian mx.  The behavior is unchanged. At first I thought it was php version hell, but its also present on php version 7.4.

Thanks.

Larry

**Comment by larrymo** [comment]
The fix of Commenting line 264 resulted in suitecrm failing to load.

However, I noticed that in my version of suitecrm  8.2.2  Line 264 DID NOT have the pair of slashes  // before the continue.   
Putting these in solved the issue, and allowed the relate fields to be shown correctly.

**Comment by scicali** [comment]
@larrymo is a workaround, not a fix, bug is open. It's a suggestion from a user (I'm not a developer!)

**Comment by lukio** [comment]
@clemente-raposo could be this ticket #251 be related?

**Comment by larrymo** [comment]
Yes this workaround works for me, although I'm not familiar enough with the
programming language to know why.  Thankyou.

**Comment by johnM2401** [comment]
Hey folks
Thank you all for raising this and your contributions.

It looks like this was resolved with the PR: 
https://github.com/salesagility/SuiteCRM-Core/pull/265
As well as other changes to Relate Fields & Enums that have been introduced since 8.2.0

For example, using a custom Relationship on Calls:

![Image](https://github.com/user-attachments/assets/75fd02b4-cd74-4acc-8f12-ac7c373d65e1)


<br>

I'm going to close this issue off for now, but please let us know if you are still experiencing this (or other issues) on recent versions of SuiteCRM.

Thank you all again!
